src/components/ui/PrimaryButton/PrimaryButton.tsx

```diff
@@ -0,0 +1,17 @@
+interface PrimaryButtonProps {
+  label: string;
+  onClick?: () => void;
+}
+
+const PrimaryButton = ({ label, onClick }: PrimaryButtonProps) => {
+  return (
+    <span
```

Comment:
If we are rendering a button I'd suggest to use the `<button>` tag.

Regarding the UI, if we add the `transition` class, the hover transition would be a bit smoother.